pyleecan/Methods/Converter/ConvertMC/convert_to_P/convert_slot_to_P.py

```diff
@@ -58,3 +81,27 @@ def convert_slot_to_P(self):
     self.get_logger().info(
         f"Conversion {slot_type} into {type(self.machine.stator.slot).__name__}"
     )
+
+    # selection type of wedge
+    if "Wedge_Model" in self.other_dict["[Winding_Design]"]:
+        wedge = self.other_dict["[Winding_Design]"]["Wedge_Model"]
+
+    else:
+        wedge = "Air"
+
+    # set the correct wedge
+    if wedge == "Air":
+        self.machine.stator.slot.wedge_mat = None
+    elif wedge == "Wedge":
+        self.machine.stator.slot.wedge_mat = Material()
+    elif wedge == "Wound_Space":
+        self.machine.stator.slot.wedge_mat = None
+        self.get_logger().info(
+            f"Conversion Wound_Space for {type(self.machine.stator.slot).__name__} has not equivalent in pyleecan or has not implement"
+        )
+        self.get_logger().info(f"we define air in place of the wedge")
+
+    else:
+        self.machine.stator.slot.wedge_mat = None
+        self.get_logger().warning(f"Error for conversion of wedge")
+        self.get_logger().info(f"we define air in place of the wedge")
```

Comment:
Error for conversion of wedge (unknow wedge type: " "). No wedge defined in the converted slot